.travis.yml

```diff
@@ -1,27 +1,23 @@
 language: python
+cache: pip
 services:
   - docker
 env:
   global:
-    - DOCKER_COMPOSE_VERSION: 1.25.5
     - APPLICATION_NAME: aquila
     - SERVICE_NAME: aquila-web
```

Comment:
Two things: 
- we've called the `SERVICE_NAME` variable `CONTAINER` in other contexts and I'd prefer to be consistent. 
- I think it makes sense to use a `DJANGO_PORT` variable here, even though there's also a environment variable already defined in Travis for deployment. Getting these ports all aligned as variables is going to take some more thinking, but in the meantime it's useful to have that pulled out as a variable so we remember we have to deal with it.